CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## Changelog
 
+### [2024.09.1](https://github.com/plus3it/spel/releases/tag/2024.09.1)
+
+**Released**: 2024.09.17
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2024.09.1/manifests>
+
+**Summary**:
+
+*   "Extra" packages updated in this release:
+    - aws-cli/2.17.33
```

Comment:
Update the version.
```suggestion
    - aws-cli/2.17.52
```

Reply:
Oops, I didn't save the file edits before committing. 😱 Fixed!